CLR_DEV9/Config/ConfigFile.cs

```diff
@@ -47,6 +51,7 @@ private void Init()
             HddEnable = false;
             DirectConnectionSettings = new ConfigDirectIP();
             SocketConnectionSettings = new ConfigSocketIP();
+            EnableLogging = new ConfigLogging();
```

Comment:
Do ConfigLogging's defaults apply correctly when loading an older config file (that lacks the entry)?

I remember having issues relating to that in the past.

Reply:
If config file lacks the entry, it applies default values. (at least on my machine)
However, if you manually add required entries, it still applies default values until you let it recreate config file. (maybe I just did it in wrong order?)

Reply:
IIRC the entries in the file have to be in a specific order for settings to be read. I'm only concerned about updating an old file, and that seems to work.